app/lib/theme/widgets/ReadOnlyWidget.tsx

```diff
@@ -0,0 +1,20 @@
+import { WidgetProps } from "@rjsf/core";
+const ReadOnlyWidget: React.FC<WidgetProps> = ({ label, value, options }) => {
+  return (
+    <>
+      {/* Individual contacts shouldn't display the label because they're grouped as either primary or secondary contacts. */}
+      {label === "Contact" ? (
```

Comment:
We want to avoid handling specific label values in this generic component. Is there a generic option to indicate whether the label should be displayed or not? This will have to be refactored with #328. The widget should only display the `dd` element